packages/grpc-native-core/test/surface_test.js

```diff
@@ -941,6 +941,18 @@ describe('Other conditions', function() {
     call.write({});
     call.end();
   });
+  it('client should not wait for ready by default', function(done) {
+    this.timeout(15000);
+    const disconnectedClient = new Client('foo.test.google.com:50051', grpc.credentials.createInsecure());
+    const metadata = new grpc.Metadata({waitForReady: false});
```

Comment:
If understanding correctly, I'm a bit of confused by the `{waitForReady: false}` argument. Shall we remove it for the **by default** test case? Thanks.

Reply:
I think you're right. It looks like that test implementation is the result of some evolution in the PR it was copied from.